### ISSUE TITLE
Execution path

### ISSUE THREAD

**Issue by lexesv** [neutralinojs/neutralinojs#372]
```
./neutralino-linux --load-dir-res app 
ERROR [src/resources.cpp:makeFileTree] resources::makeFileTree error: res.neu is missing.
```
OK. Working.

```
./bin/neutralino-linux --load-dir-res app 
ERROR [src/resources.cpp:makeFileTree] resources::makeFileTree error: res.neu is missing.
ERROR [src/settings.cpp:getSettings] std::exception
terminate called after throwing an instance of 'nlohmann::detail::type_error'
  what():  [json.exception.type_error.302] type must be string, but is null
Aborted
```
Error

```
./bin/neutralino-linux --load-dir-res ./bin/app
ERROR [src/resources.cpp:makeFileTree] resources::makeFileTree error: res.neu is missing.
ERROR [src/settings.cpp:getSettings] std::exception
terminate called after throwing an instance of 'nlohmann::detail::type_error'
  what():  [json.exception.type_error.302] type must be string, but is null
```
Error

Question: how to run the neutralino not from the directory in which it is located

**Comment by shalithasuranga** [comment]
Looks like res.neu is loaded by using the path that is relative to the binary. We need to fix this.

**Comment by lexesv** [comment]
>Looks like res.neu is loaded by using the path that is relative to the binary

I don't use res.neu

**Comment by lexesv** [comment]
neutralino also doesn't run via a symbolic link

**Comment by shalithasuranga** [comment]
Hey.. this is fixed. will be released with v1.9.0. Now we have NL_PATH global variable with the relative application path. Also can be launched from another directory. We will check the symlink/shortcuts issue with https://github.com/neutralinojs/neutralinojs/issues/375

**Comment by uAliFurkanY** [comment]
Hey, this fix does not cover the original use case, providing an execution path with **arguments**.  
One should be able to use `--load-dir-res` without editing neutralino.config.json
